iothub_client/samples/iothub_convenience_sample/iothub_convenience_sample.c

```diff
@@ -240,9 +241,14 @@ int main(void)
 
         // Set any option that are neccessary.
         // For available options please see the iothub_sdk_options.md documentation
+#ifndef SAMPLE_HTTP
+        bool traceOn = true;
+        (void)IoTHubDeviceClient_SetOption(device_handle, OPTION_LOG_TRACE, &traceOn);
+#endif
+        // ms_delay can be between 1 and 100 milliseconds
```

Comment:
Please describe what this does or link to docs that do.

Reply:
May be add a comment for this ifndef to make it more clear.
// Can not set this options in HTTP